Source/Core/Core/HW/SI/SI_DeviceGCController.cpp

```diff
@@ -33,6 +34,14 @@ CSIDevice_GCController::CSIDevice_GCController(SIDevices device, int device_numb
   m_origin.origin_stick_y = GCPadStatus::MAIN_STICK_CENTER_Y;
   m_origin.substick_x = GCPadStatus::C_STICK_CENTER_X;
   m_origin.substick_y = GCPadStatus::C_STICK_CENTER_Y;
+
+  m_config_changed_callback_id = Config::AddConfigChangedCallback([this] { RefreshConfig(); });
+  RefreshConfig();
+}
+
+CSIDevice_GCController ::~CSIDevice_GCController()
```

Comment:
Note sure why lint didn't catch this, but:

```suggestion
CSIDevice_GCController::~CSIDevice_GCController()
```